app/javascript/components/App.js

```diff
@@ -12,18 +15,17 @@ class App extends Component {
     } = this.props;
     return (
       <>
-        <Header />
-        {logged_in && (
-          <div>
-            <a href={sign_out_route}>Sign Out</a>
-            <CoinIndex />
-          </div>
-        )}
-        {!logged_in && (
-          <div>
-            <a href={sign_in_route}>Sign In</a>
-          </div>
-        )}
+        <Router>
+          <Header {...this.props} />
+          <Switch>
+            {/* <Route exact path="/" component={Home} /> */}
+            {/* //<Route path="/catindex" component={CatIndex} /> */}
+            {/* //<Route path="/catshow" component={CatShow} /> */}
+            {/* <Route path="/users/sign_in" component={SignIn} /> */}
+            {/* //<Route path="/catedit" component={CatEdit} /> */}
+            {/* //</Switch></Router>Route component={NotFound} /> */}
+          </Switch>
```

Comment:
You may want to remove the cat routes here. Definitely remove the `SignIn` route.  And looks like there are some extra tags on line 26.

Reply:
Will do. I should have cleaned those things up yesterday as I was doing this. I will knock it out. Thank you.